gpfit/tests/t_print_fit.py

```diff
@@ -1,49 +1,61 @@
 "unit tests for gpfit.print_fit module"
 import unittest
 from numpy import array
-from gpfit.print_fit import print_MA, print_SMA, print_ISMA
+from gpfit.print_fit import print_ma, print_sma, print_isma
 
 A = array([2, 3, 4, 6, 7, 8, 10, 11, 12])
 B = array([1, 5, 9])
-ALPHA = array([1./13., 1./14., 1./15.])
+ALPHA = array([1.0 / 13.0, 1.0 / 14.0, 1.0 / 15.0])
```

Comment:
wow that's a little extra. especially for py3!

Reply:
Ha yeah, missed this. Will fix to `1/13` etc.